.github/workflows/integration_test.yml

```diff
@@ -24,7 +24,8 @@ jobs:
 
       - name: Install Chaos Mesh
         run: |
-          bash install.sh --local kind --crd ./manifests/crd.yaml
+          export CLUSTER="chart-testing"
```

Comment:
the default cluster name of the k8s  created by `helm/kind-action`

Reply:
I think we also use this CI to test `install.sh`. So if we merge this pr, `install.sh` won't be tested.

Reply:
so we can add another ci to use the `hack/local-up-chaos-mesh.sh`? I think it's ok